core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2597,6 +2605,22 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
   }
 
   /**
+   * Checks whether the owner belongs to the group.
+   *
+   * @param owner the owner to check
+   * @param group the group to check
+   * @throws FailedPreconditionException if owner does not belong to group
+   */
+  private void checkUserBelongsToGroup(String owner, String group)
+      throws IOException {
```

Comment:
I don't feel strongly either way about putting `FailedPreconditionException` in the signature. Putting it there would serve as a reminder to callers that the method throws `FailedPreconditionException` in case they want to catch it explicitly, but the nature of this method makes it unlikely that anyone will ever want to catch `FailedPreconditionException` specifically, so I'm also fine with leaving it out of the signature.

We don't javadoc `IOE` because the javadoc is always `@throws IOException if an IO error occurs`. The purpose of `IOE` is to indicate that the method is expected to fail sometimes and we should handle those situations gracefully.